herddb-cli/src/main/java/herddb/cli/HerdDBCLI.java

```diff
@@ -1595,23 +1607,32 @@ private static void println(Object msg) {
         System.out.println(msg);
     }
 
-    private static void runSqlConsole(Connection connection, Statement statement, boolean pretty, boolean verbose) throws IOException {
+    private static void runSqlConsole(Statement statement, boolean pretty, boolean verbose, boolean persistHistory) throws IOException, SQLException {
         Terminal terminal = TerminalBuilder.builder()
                 .system(true)
                 .build();
-        LineReader reader = LineReaderBuilder.builder()
+        LineReaderBuilder readerBuilder = LineReaderBuilder.builder()
                 .history(new DefaultHistory())
-                .terminal(terminal)
-                .build();
-        String prompt = "herd: ";
+                .terminal(terminal);
+        if (persistHistory) {
+            String historyDirectory = System.getProperty("user.home", ".");
+            File historyFile = new File(historyDirectory, ".herddb.cli.history");
```

Comment:
How long can this file grow?

Reply:
It is unlimited 
We could set a bound 

https://www.javadoc.io/doc/org.jline/jline/3.5.1/org/jline/reader/LineReader.html#HISTORY_FILE_SIZE

Reply:
I have added 100 as bound.